ParseCAN/spec/bus/type.py

```diff
@@ -154,24 +149,19 @@ def ctype(self) -> str:
 
         return self.type
 
-    def dtype(self):  # -> np.dtype:
-        raise NotImplementedError('not updated yet')
-        # if self.isenum():
-        #     '''
-        #     Return a unicode string with length equal to the maximum possible
-        #     length of any of the contained value names.
-        #     '''
-        #     return 'U' + str(max(len(val.name) for val in self.enum))
+    def dtype(self): # -> np.dtype:
+        # NOTE: This may have bugs. Test more robstly
```

Comment:
It seems OK given that all the allowed type strings map to a dtype.

Reply:
Note to self: We should likely make an enum of the valid types instead of `valid_types`.